app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -333,7 +334,15 @@ public void displayCurrentInfo() {
                     currentBgValueText.setText(stringEstimate + " " + BgReading.slopeArrow());
                 }
             }
-            if (bgGraphBuilder.unitized(estimate) <= bgGraphBuilder.lowMark) {
+            int minutes = (int)(System.currentTimeMillis() - lastBgreading.timestamp) / (60 * 1000);
+            notificationText.append("\n" + minutes + ((minutes==1)?" Minute ago":" Minutes ago"));
+            List<BgReading> bgReadingList = BgReading.latest(2);
+            if(bgReadingList != null && bgReadingList.size() == 2) {
+                // same logic as in xDripWidget (refactor that to BGReadings to avoid redundancy / later inconsistencies)?
+                notificationText.append("\n"
+                        + bgGraphBuilder.unitizedDeltaString(lastBgreading.calculated_value - bgReadingList.get(1).calculated_value));
+            }
+            if(bgGraphBuilder.unitized(estimate) <= bgGraphBuilder.lowMark) {
```

Comment:
can you use BgReading.getXRecentPoints(2)
This function also checks that the 2 points are from relatively close times, so it will not display delta, if one point is recent and the other is from 3 hours ago.